evm_loader/program/src/account_data.rs

```diff
@@ -136,7 +136,7 @@ impl AccountData {
 }
 
 impl Account {
-    const SIZE: usize = 20+1+8+32+32+1+32;
+    pub const SIZE: usize = 20+1+8+32+32+1+32;
```

Comment:
Why you want to make it public? You don't use it.

Reply:
`SIZE` used in https://github.com/neonlabsorg/solana-program-library/pull/169/files#diff-2d7825462cfe18097d90eff351f21b30572b259b71f90a9fa5c1256f53da87c2R375